packages/react/package.json

```diff
@@ -34,7 +34,7 @@
   "scripts": {
     "prebuild": "rimraf dist",
     "build": "yarn run build:catalog && yarn run build:ts",
-    "build:ts": "rollup --config",
+    "build:ts": "NODE_OPTIONS=--max_old_space_size=3072 rollup --config",
```

Comment:
We need to bump up Node memory for build. Default allocated size for Node is 2GB in CI <- https://github.com/aws-amplify/amplify-ui/actions/runs/3691169033/jobs/6248977217